app/controllers/api_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ApiController < ActionController::Base
```

Comment:
This is great! But I don't see where you are using it 🤔 This approach is very common on APIs, but you have to change your controllers to inherit from this class instead of ApplicationController